ohttp/Cargo.toml

```diff
@@ -9,11 +9,13 @@ description = "Oblivious HTTP"
 repository = "https://github.com/martinthomson/ohttp"
 
 [features]
-default = ["client", "server", "rust-hpke"]
+default = ["client", "server", "rust-hpke-nopq"]
 client = []
 server = []
 nss = ["bindgen"]
-rust-hpke = ["hpke/x25519", "rand", "aead", "aes-gcm", "chacha20poly1305", "hkdf", "sha2"]
+rust-hpke-nopq = ["rust-hpke", "hpke"]
+rust-hpke-pq = ["rust-hpke", "hpke-pq"]
```

Comment:
My preference is for a single, optional feature here: pq (or maybe "kyber-preview").  We can then add some code:

```rust
#[cfg(all(pq, nss))]
compile_error!("NSS does not support Kyber");
```

(NSS is growing support for Kyber as well, so this should be temporary.)

Reply:
I agree. Unfortunately I didn't get it to work.

Reply:
What did you try?  It should be possible to have something as simple as this:

```toml
pq = [ "hpke-pq" ]
rust-hpke = [ "hpke", ... ]
```

That will lead to the HPKE PQ crate being loaded for compilation when "nss" and "pq" is enabled or the HPKE base crate being loaded for compilation when "rust-hpke" and "pq" is enabled, but I can live with that.  The code won't need to be linked, so the cost is only in build times and it should be possible to iron that out once things stabilize and you upstream the PQ changes.

Reply:
If you're happy with compiling unnecessary crates, then this will indeed work.